agent/provider/src/payments/payments.rs

```diff
@@ -567,6 +577,12 @@ impl Handler<ActivityDestroyed> for Payments {
         agreement.activity_destroyed(&msg.activity_id).unwrap();
 
         let payment_model = agreement.payment_model.clone();
+        let last_payable_debit_node = match agreement.payment_timeout {
+            // Ensure that last debit note is always payable, by
```

Comment:
comment not finished:(

if this is the only issue I will update this in a follow up PR so we can build an RC for SDK team